ssv/docs/README.md

```diff
@@ -0,0 +1,54 @@
+# Documentation
+
+
+<p align="center",float="left">
+<img src="./simple_class_diagram.drawio.png", width="70%" height="10%">
+</p>
+
+## Network
+
+`Network` is an interface that allows the broadcasting of a message to the P2P network through its `Broadcast` method.
+
+## BeaconNode
+
+`BeaconNode` is an interface that allows the node to fetch duty data (e.g. an Attestation vote and a block to be proposed) and to submit a signed data (e.g. a signed Attestation vote and a signed block).
+
+It also provides a method to get the blockchain domain (a sequence of bytes used in signatures that identifies the blockchain) and it holds a `BeaconNetwork` that provides information about the blockchain such as the current slot and fork version.
```

Comment:
identifies the blockchain -> identifies the blockchain, eg. mainnet / holesky / ...